src/Propel/Generator/Builder/Om/AbstractOMBuilder.php

```diff
@@ -245,6 +245,17 @@ public function getNamespace(): ?string
         return $this->getTable()->getNamespace();
     }
 
+    /**
+     * Returns the user-defined namespace for this table,
+     * or the database namespace otherwise.
+     *
+     * @return string Currently returns null in some cases - should be fixed
+     */
+    public function getNamespaceOrFail(): string
```

Comment:
`@return string|null Currently returns null in some cases - should be fixed`
says already that there is sth a bit weird underneath.

But I am not sure if we can fix those 2 methods here though without heavy bc breaks to userland code
Apparently, they can already have implemented object creation with nullable return possible.
So maybe we try to get the API as clean as possible with this PR and future ones could then maybe aim to fix up the superfluous ones (we can also deprecate again where needed).

Right now we have very limited capacity but huge amount of work ahead.

This way in PR seems both safer and less problematic than the alternative of just making it not null directly (changing behavior of existing one).